src/calculateRentalCost.js

```diff
@@ -29,6 +29,15 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const rentDay = 40;
+
+  if (days >= 1 && days < 3) {
```

Comment:
```suggestion
  if (days < smallDiscountPeriod) {
```
Вынеси числовые значения в переменные